expose-daemon.ps1

```diff
@@ -0,0 +1,149 @@
+$ErrorActionPreference = 'Stop';
```

Comment:
This whole file really seems like it shouldn't be in toplevel dir

Reply:
For the moment I don't have a great place for it. Even `bin` doesn't seem like a great place. Maybe it and other files like `parse-changelog.sh` and `release.sh` can live in a newly created directory called `ci`. But for the moment I think its in good company.

Reply:
It just seems so out of place with other project-level files that I still believe that it really shouldn’t be here - it's specific to:
- CI builds 
- A single subset of environments
- A CI-specific version of that subset

Reply:
I'll make a follow up issue for a `ci` directory to contain this and other ci scripts

Reply:
#227